styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+ul,
+a {
+  text-decoration: none;
+  list-style: none;
+}
+
+.container {
+  margin: auto;
+  width: 375px;
+  height: 718px;
+  border: 1px solid rebeccapurple;
+  border-bottom-left-radius: 7%;
+}
+
+aside ul {
+  display: flex;
+  padding-top: 10px;
+}
+
+.header ul {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px 10px 12px 40px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 8px 30px;
+}
+
+.logo {
+  width: 100px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.background-img {
+  background-image: url(./assets/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  height: auto;
+}
+
+.writer {
+  padding: 80px 20px 0 12px;
+  width: 330px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 56px;
+  line-height: 55px;
```

Comment:
- [ ] Kindly make sure all your text uses the right `font-family` as seen in your chosen Figma design 🙏 . This would help make your work look as close as possible to the Figma design ☺️ 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/198428452-6d53bf90-a5f2-4420-a331-1f3ac3e29aa4.png)  |  ![image](https://user-images.githubusercontent.com/74814780/198428221-2723971b-b14d-4f6e-a156-bd6950040158.png)